src/App.js

```diff
@@ -1,26 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+import React, {useEffect} from 'react';
+import styled, {ThemeProvider} from 'styled-components';
+import {AppTheme, lightTheme} from '@datorama/app-components';
+import {Provider, useDispatch, useSelector} from 'react-redux';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import sdk from '@datorama/sdk';
+import Install from "./component/Install";
+import Run from "./component/Run";
+import store from './redux/store';
+import {saveSession} from "./redux/actions/session.actions";
+import {isEmpty} from "lodash";
+
+// pages
+
+
+const App = () => {
+    const dispatch = useDispatch();
+    const session = useSelector(state => state.session);
+
+    useEffect(() => {
+        sdk.init();
```

Comment:
this approach is good, lately we work with hooks. and we create a custom hook called useInit for example - 


const useInit = () => {
  const [config, setConfig] = useState();

  useEffect(() => {
    sdk.init();

    sdk.app.onLoad().then(serverConfig => {
        setConfig(serverConfig);
    }, []);
  return config;
};

Reply:
and here for example we can also dispatch an action to save the session in redux